CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ ___
 - IMPROVE: Added new account lockout policy option `accountLockout.unlockOnPasswordReset` to automatically unlock account on password reset. [#7146](https://github.com/parse-community/parse-server/pull/7146). Thanks to [Manuel Trezza](https://github.com/mtrezza).
 - IMPROVE: Parse Server is from now on continuously tested against all recent MongoDB versions that have not reached their end-of-life support date. Added MongoDB compatibility table to Parse Server docs. [7161](https://github.com/parse-community/parse-server/pull/7161). Thanks to [Manuel Trezza](https://github.com/mtrezza).
 - IMPROVE: Parse Server is from now on continuously tested against all recent Node.js versions that have not reached their end-of-life support date. [7161](https://github.com/parse-community/parse-server/pull/7177). Thanks to [Manuel Trezza](https://github.com/mtrezza).
+- IMPROVE: Cloud Validators will now log on invalid configuration [#7154](https://github.com/parse-community/parse-server/pull/7154). Thanks to [dblythy](https://github.com/dblythy)
```

Comment:
I have seen multiple names for this feature so far:
- cloud function validation
- Cloud Validators
- Parse.Cloud validator

Let's find a consistent name; it is validating `Cloud Functions` that are part of `Cloud Code`; in the docs this is already titled "cloud function validation", so let's stick with `Cloud Function validation`. How about:

> - IMPROVE: Log error on invalid Cloud Function validation configuration.